CHANGELOG.md

```diff
@@ -1,3 +1,12 @@
+# 2.0.0
+
+### Feature
+
+- [#155](https://github.com/okta/okta-react-native/pull/155) Adds the promise for `signOut` and `signInWithBrowser` methods (iOS)
```

Comment:
Let's use `Changes` instead of `Adds` for breaking changes. `Adds` reads like adding a new feature. The log can be something like, "Changes `signOut` method signature to return a promise"